components/base-node.js

```diff
@@ -44,6 +46,18 @@ module.exports = {
    },
 
   /**
+   * Standard component error handling - send a message to the error port and throw Error
+   *
+   * @param message message to be sent to error port
+   * @param errorPort handle to the component's error port
+   */
+   handleError: function( message, errorPort ) {
```

Comment:
I don't see the point of this function, whenever I test it it causes the error to be reported twice.